File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/numar-sedinte-comisie-gl-report.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { NumarSedinteComisieGlReportRowModel } from "./numar-sedinte-comisie-gl-report-row.model";

@JsonObject
export class NumarSedinteComisieGlReportModel {
	
	@JsonProperty("rows", [NumarSedinteComisieGlReportRowModel])
	public rows: NumarSedinteComisieGlReportRowModel[] = [];
	
	@JsonProperty("totalComisie", Number)
	public totalComisie: number = null;
	
	@JsonProperty("totalGl", Number)
	public totalGl: number = null;
	
	@JsonProperty("totalGeneral", Number)
	public totalGeneral: number = null;
    
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NumarSedinteComisieGlReportModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class NumarSedinteComisieGlReportModel {

	private List<NumarSedinteComisieGlReportRowModel> rows;
	private Integer totalComisie;
	private Integer totalGl;
	private Integer totalGeneral;

	public List<NumarSedinteComisieGlReportRowModel> getRows() {
		return rows;
	}

	public void setRows(List<NumarSedinteComisieGlReportRowModel> rows) {
		this.rows = rows;
	}

	public Integer getTotalComisie() {
		return totalComisie;
	}

	public void setTotalComisie(Integer totalComisie) {
		this.totalComisie = totalComisie;
	}

	public Integer getTotalGl() {
		return totalGl;
	}

	public void setTotalGl(Integer totalGl) {
		this.totalGl = totalGl;
	}

	public Integer getTotalGeneral() {
		return totalGeneral;
	}

	public void setTotalGeneral(Integer totalGeneral) {
		this.totalGeneral = totalGeneral;
	}

}
